src/frontend/e2e/utils/schemas/sampleResponseData.ts

```diff
@@ -35,4 +26,30 @@ export type SampleResponseData = {
     id: number;
     name: string;
   };
+  lineages: [
+    {
+      lineage_type?: string;
+      lineage?: string;
+      last_updated: string;
+      lineage_software_version?: string;
+      lineage_probability?: number;
+      reference_dataset_name?: string;
+      reference_sequence_accession?: string;
+      reference_dataset_tag?: string;
+      scorpio_call?: string;
+      scorpio_support?: string;
+      qc_status?: string;
+    }
+  ];
+  qc_metrics: [
+    {
+      qc_score?: string;
+      qc_software_version?: string;
+      qc_status?: string;
```

Comment:
for the brief period of time when a user uploads new samples and the nextclade job is runnign the samples may not have the associated lineage/qc_metrics right?

Reply:
Yeah, but they wouldn't have anything in here then, right? It's an array of objects, but if the object exists in the array that key should have a guaranteed value. I don't think it's possible for the lineage/qc_metric to exist but that value to be missing from it. Either it's an empty list or it'll be there.

Reply:
yeah good point, i'll go back and add not optional to some of these, for some reason i was thinking they needed to be optional if the whole object could not exist but that was the wrong thinking.

Reply:
wait - the backend supports multiple lineage/qc types, and so the frontend should too, right? I also don't think the frontend should grab the first element in the list - it can use a feature flag to figure out which lineage/qc call is the default and use that.